src/plugins/navigation/public/top_nav_menu/create_top_nav_menu.tsx

```diff
@@ -25,7 +28,12 @@ export function createTopNav(
 
     return (
       <I18nProvider>
-        <TopNavMenu {...props} unifiedSearch={unifiedSearch} config={config} />
+        <TopNavMenu
+          {...props}
+          unifiedSearch={unifiedSearch}
+          config={config}
+          minRefreshInterval={minRefreshInterval}
```

Comment:
This seems odd to me... Can we make the navigation & `TopNavMenu` agnostic of this property, and have unified search provide it?

Reply:
44e5bbd